cli/connectivity.go

```diff
@@ -238,3 +216,31 @@ func registerCommonFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&params.TestNamespace, "test-namespace", defaults.ConnectivityCheckNamespace, "Namespace to perform the connectivity in")
 	flags.Var(&params.DeploymentAnnotations, "deployment-pod-annotations", "Add annotations to the connectivity pods, e.g. '{\"client\":{\"foo\":\"bar\"}}'")
 }
+
+func newConnectivityTests(params check.Parameters) ([]*check.ConnectivityTest, error) {
+	if params.TestConcurrency < 1 {
+		fmt.Printf("--test-concurrency parameter value is invalid [%d], using 1 instead\n", params.TestConcurrency)
+		params.TestConcurrency = 1
+	}
+	if params.TestConcurrency < 2 {
+		cc, err := check.NewConnectivityTest(k8sClient, params, defaults.CLIVersion)
+		if err != nil {
+			return nil, err
+		}
+		return []*check.ConnectivityTest{cc}, nil
+	}
```

Comment:
Rather than have a special case path for `TestConcurrency == 1` which is identical to the loop body, why not remove this block entirely?

Reply:
In case of `params.TestConcurrency < 2` it doesn't change `params.TestNamespace`.

Reply:
Does anything rely on the pre-existing namespace name? I'm fine with this as is, but it look like an opportunity to remove a few chunks of code.

Reply:
The namespace changing from `cilium-test` to `cilium-test-1`, doesn't seem too confusing. This change can be left for the future, up to you.